openlibrary/components/MergeUI/MergeTable.vue

```diff
@@ -113,6 +113,26 @@ export default {
             return records;
         },
 
+        async enhancedRecords(){
+            if (!this.records) return null;
+
+            const author_names = await get_author_names(this.records)
```

Comment:
...to a try catch that just returns the clone without any modifications. Otherwise an error here will cause the entire UI to fail!